test/users/engin/compiler/parentheseslessRecConst.chpl

```diff
@@ -0,0 +1,20 @@
+record independentRecord { var field = 5;} 
+
+var ir = new independentRecord;
+writeln(ir.field);
+//above lines compile with/without --verify
```

Comment:
I think that makes sense( unless there are parenthesesless constructors). Maybe I should simplify this case and change it to an error message future rather than a bug.